ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_building_matrices.py

```diff
@@ -161,3 +162,37 @@ def test_numpy_vander(
         N=N,
         increasing=increasing,
     )
+
+
+# diagflat
+@handle_frontend_test(
+    fn_tree="numpy.diagflat",
+    args_packet=_generate_diag_args(),
+)
+def test_numpy_diagflat(
+    *,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+    args_packet
+):
+    dtype_x, offset, dtype_padding_value, align, num_rows, num_cols = args_packet
+
+    x_dtype, x = dtype_x
+    padding_value_dtype, padding_value = dtype_padding_value
+    padding_value = padding_value[0][0]
+
+    helpers.test_frontend_function(
+        input_dtypes=x_dtype + ["int64"] + padding_value_dtype,
+        test_flags=test_flags,
+        on_device=on_device,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        v=x[0],
+        offset=offset,
```

Comment:
here should only be `v` and `k` after that

Reply:
I have done that but I got an error `E  ivy.utils.exceptions.IvyBackendException: numpy: diagflat: diagflat() got an unexpected keyword argument 'k'`